.github/workflows/build.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-release
+  cancel-in-progress: true
+
+jobs:
+  pr-builder:
+    needs:
+      - checks
+      - build-and-test-wheels
+    secrets: inherit
+    uses: rapidsai/shared-action-workflows/.github/workflows/pr-builder.yaml@branch-23.12
```

Comment:
This repo was renamed to https://github.com/rapidsai/shared-workflows. Please update `shared-action-workflows` to `shared-workflows` accordingly.

Reply:
Also let's use `branch-24.02` since 23.12 is in freeze. We can go ahead and target this as if it's being released with 24.02.

Reply:
updated